server/engine/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/replacement/SearchPattern.java

```diff
@@ -74,10 +100,17 @@ private static boolean isQuotedString(String text) {
   // Parentheses { ( } ... {  //NOSONAR
   // ) }, Colon { : }  //NOSONAR
   // Ref - https://www.ibm.com/support/knowledgecenter/SS6SG3_6.2.0/lr/ref/rllanrul.html
-  private static final String SEPARATOR_REGEX_SUFFIX = "(?=[\\):]|[,;]\\s|\\.\\s*|\\s|$)[\\):,;]?";
-  private static final String SEPARATOR_REGEX_PREFIX = "(\\(|:|[,;]\\s)?(?<=^|[.,;]\\s|\\s|[\\(:])";
-  private static final List<String> COBOL_SEPARATORS_START = ImmutableList.of(";", "(", ":", ",");
-  private static final List<String> COBOL_SEPARATORS_END = ImmutableList.of(";", ")", ":", ",");
+  private static final String SEPARATOR_REGEX_SUFFIX = "(?=[\\):]|[,;]\\s|\\.\\s*|\\s|$)";
+  private static final String SEPARATOR_REGEX_PREFIX = "(?<=^|[.,;]\\s|\\s|[\\(:])";
```

Comment:
- I don't think `\` is a valid separator (in either rule).
- Are some of the `\s` spaces really restricted to a single space? Should the rule be `\s+` instead?